particula/util/rms_speed.py

```diff
@@ -0,0 +1,35 @@
+""" calculating rms speed of molecules
+"""
+
+import numpy as np
+from particula.constants import GAS_CONSTANT, MOLECULAR_WEIGHT_AIR
+from particula.util.input_handling import in_molecular_weight, in_temperature
+
+
+def cbar(**kwargs):
+    """ Returns the mean speed of molecules in an ideal gas.
+
+        Parameters: (either # or $)
```

Comment:
what does either # or $ mean?

Reply:
These were references for me (one of them means some constant is default and the other is utility is a default). Anyway, please don't merge this --- you can "approve" it, but leave it for me to merge later.

Reply:
Because I don't know how much of it will make sense in before the release... I will get back to it after the release. I don't know how to make this a draft PR again 😶‍🌫️